access/types/forms.py

```diff
@@ -230,19 +238,6 @@ def add_field(self, i, config, field_class, widget_class,
         if self.disabled:
             args['widget'].attrs['disabled'] = 'disabled'
 
-        if args['required'] and (
-            widget_class is not forms.CheckboxSelectMultiple or len(choices) == 1
-        ):
-            # The required HTML attribute should not be set to a question with
-            # multiple checkboxes since the browser would require that the user
-            # selects every checkbox.
-            # Note: newer Django versions include the required HTML attribute
```

Comment:
I can't see anywhere that the Django version would have been upgraded in the MOOC-Grader. If it has not been upgraded, then this if branch shouldn't be deleted, right?

Reply:
Hmm.. then I don't know what has happened in between writing those comments and now. Since it seems to work as expected now without that extra attribute set. 

The browser does not require the user to check every checkbox, just one. And the validation happens when the form is being assessed, so the appearance is correct and consistent for checkboxes without answers:

![image](https://user-images.githubusercontent.com/43036333/90227068-a6f8df80-de1c-11ea-89e1-cf9e75cf195e.png)

Reply:
Verify that the html attribute `required` is not defined for input checkbox.

Reply:
Doesn't seem to be: 
![image](https://user-images.githubusercontent.com/43036333/90242384-99038880-de35-11ea-8992-2e090b0cdda8.png)

Reply:
The current versions of Chrome and Firefox I have work perfectly

Reply:
Accessibility guidelines don't really recommended using the browser built-in validation with the `required` HTML attribute, so even if the attribute is not included now when the code is removed, we can accept that.